src/parser/index.js

```diff
@@ -16,6 +13,7 @@ function componentParser(templatePath: string, defaults: Object, type: string):
         if (cachedComponentContent) {
             parseContent(cachedComponentContent, templatePath, defaults, type)
                 .then(contentObject => {
+                    //TODO I think we should move the cache in here
```

Comment:
this is where i think we should move the cache too..

so we can skip parse content, we would have to do a merge on the data objects but thats it